### ISSUE TITLE
Segmentation fault when using roipool3d_cuda

### ISSUE THREAD

**Issue by JenningsL** [sshaoshuai/PointRCNN#10]
Hi, thank you for sharing. I am trying to run generate_gt_database.py. But I got Segmentation fault when `roipool3d_cuda.pts_in_boxes3d_cpu` is called. Is it related to the CUDA version? May I ask which version of CUDA you are using?

**Comment by sshaoshuai** [comment]
I have passed the testing with both CUDA 8.0 and CUDA 9.0, and this function is a cpu based function. 
Maybe you should pay attention to the compiling logs to find whether there are some problems.

**Comment by JenningsL** [comment]
I only found some warning about the g++ version. No error.
```
                              !! WARNING !!

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Your compiler (g++ 4.8.5) may be ABI-incompatible with PyTorch!
Please use a compiler that is ABI-compatible with GCC 4.9 and above.
```

**Comment by sshaoshuai** [comment]
Yes, this is the reason of segmentation fault. 
You should use GCC 4.9+ as it said, please update your GCC version. 
See here [https://github.com/sshaoshuai/Pointnet2.PyTorch/issues/1](https://github.com/sshaoshuai/Pointnet2.PyTorch/issues/1) for more.

**Comment by JenningsL** [comment]
Thanks, problem solved after upgrading GCC to 5.2.0.
